docs/user/dql/functions.rst

```diff
@@ -139,6 +139,20 @@ Description
 
 Specification is undefined and type check is skipped for now
 
+
+CONV
+====
+
+Description
+-----------
+
+Usage: CONV(x, a, b) converts the number x from a base to b base
+
+Argument type: x: string, a: integer, b: integer
+
+Return type: string
```

Comment:
Could you add example for each function? Reference: https://dev.mysql.com/doc/refman/8.0/en/mathematical-functions.html#function_mod

For example,

```
    od> SELECT MOD(5, 2), MOD(-5, 2), MOD(5, 0);
    fetched rows / total rows = 1/1
    +---------+-------------+-------------+
    ...
```

Reply:
Sorry I forgot to add examples here. Will do